fiftyone/utils/data/exporters.py

```diff
@@ -1708,7 +1708,7 @@ def log_collection(self, sample_collection):
         # Exporting the information below only makes sense when exporting an
         # entire dataset
 
-        if dataset.has_views:
+        if dataset.has_saved_views:
```

Comment:
Should rename `"views"` to `"saved_views"` in these importer/exporter JSON too, for consistency

Reply:
should `export_views` then be `export_saved_views` and likewise for imports or do you think it's fine without prefixing with `saved`

Reply:
Hmm, well we're in it this far, so I guess so right!?!

Reply:
fwiw I did go through this thought process when I initially added this stuff, and at that time I chose my personal preference of going for as few keystrokes as possible (so `view` not `saved_view`). But, if our stance is that users may get confused seeing just "view" since this is not referring to any old view they may have created, then I would go with principal #2 instead: be super consistent on public APIs

Reply:
yeah I think consistency is best here, especially since most people will have autocomplete to minimize the keystrokes anyways